lib/runtime-core/src/export.rs

```diff
@@ -26,6 +28,8 @@ pub enum Export {
 #[derive(Debug, Clone)]
 pub struct FuncPointer(*const vm::Func);
 
+unsafe impl Send for FuncPointer {}
```

Comment:
For this in particular, I think it's relevant that we don't have any thread-local variables.

Reply:
Hmm, can you explain what you mean?  I agree that thread-local variables can't be Send, is that something we've done or talked about in the past with FuncPointer?